openmetadata-spec/src/main/resources/json/schema/email/smtpSettings.json

```diff
@@ -52,5 +52,5 @@
     }
   },
   "additionalProperties": false,
-  "required": ["serverEndpoint", "serverPort", "username", "password", "senderMail"]
+  "required": ["serverEndpoint", "serverPort", "username", "password", "senderMail", "openMetadataUrl"]
```

Comment:
`openMetadataUrl` looks like we are duplicating this value in different parts of the configuration.